app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/ui/TrackingProtectionExclusionListActivity.kt

```diff
@@ -264,7 +264,6 @@ class TrackingProtectionExclusionListActivity :
 
     override fun onDefaultProtectionRestored() {
         viewModel.restoreProtectedApps()
-        restartVpn()
```

Comment:
Removed the `restartVpn()` call as `viewModel.restoreProtectedApps()` already restarts the vpn.